components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -614,11 +614,13 @@ const ContributorHighlightCard = ({
       </div>
 
       {/* Generated OG card section */}
-      {type === "pull_request" || type === "issue" ? (
-        <GhOpenGraphImg githubLink={highlightLink} />
-      ) : (
-        <DevToSocialImg blogLink={highlightLink} />
-      )}
+      <a href={highlightLink} target="_blank" rel="noreferrer" aria-hidden="true">
```

Comment:
`aria-hidden` is used here so that a screen reader or other assistive technology doesn't need to see a repeat of the same link.